sequential/maps/src/solution/maps_to_records.erl

```diff
@@ -0,0 +1,10 @@
+-module(maps_to_records).
```

Comment:
why do you use a module for each function instead of single module with all of them?